src/main/java/com/checkmarx/flow/controller/GitHubController.java

```diff
@@ -342,6 +342,7 @@ public String pingRequest(
                 }
             }
             emails.add(event.getPusher().getEmail());
+            emails.add(event.getRepository().getOwner().getEmail().toString());
```

Comment:
is there a chance that one of the methods returns null?
event.getRepository().getOwner().getEmail()
if it could happen, there might be a nullpointerexception when accessing to the next method

Reply:
Commit a4afeb7